core/src/main/java/org/lint/azzert/context/TestMethodContext.java

```diff
@@ -30,14 +31,16 @@ public TestMethodContext(TestMethodContext source) {
         this.visible = source.visible;
         this.annotations = new HashSet<>(source.annotations);
         this.assertMethodsAtLineNumbers = new LinkedList<>(source.assertMethodsAtLineNumbers);
+        this.ignored = source.ignored;
     }
 
     public void resetMethodDetails() {
         this.methodName = "";
         this.methodSignature = "";
         this.annotations = new HashSet<>();
-        this.assertMethodsAtLineNumbers  = new LinkedList<>();
+        this.assertMethodsAtLineNumbers = new LinkedList<>();
```

Comment:
Could you please honor the existing code format in all places?

Reply:
Sure